### ISSUE TITLE
Markdown rendering and linefeeds

### ISSUE THREAD

**Issue by millette** [beakerbrowser/beaker#520]
Operation System: Debian Jessie
Beaker Version: 0.7.2

It appears linefeeds are rendered in markdown rendering. For instance, the following is written on 3 lines (2 words per line), but it should render as a single paragraph:

So there is more to come.

Instead, it's rendering as

So there
is more
to come.

**Ahum** Looks like github is also displaying the same behavior... I have always used 2 linefeeds to separate paragraphs in markdown. There goes another assumption... The [Markdown GitHub docs](https://help.github.com/articles/basic-writing-and-formatting-syntax/#paragraphs-and-line-breaks) tell a different story, too.

**Comment by pfrazee** [comment]
Yeah this is sort of contentious. I'm actually a fan of the current behavior, because it enables `<br>`s to exist, and sometimes I want linebreaks without the added margin of a `<p>`.

**Comment by pmario** [comment]
>Instead, it's rendered as
>
>So there
is more
to come.

This is GFM github flavored markdown. "pure markdown" would render it as a single paragraph. 

May be commonmark will get more traction in the future: http://spec.commonmark.org/0.26/#hard-line-breaks ... They have a different approach again :/ ... Time will tell, who wins. 

@millette .. I wouldn't rely on beaker to render your stuff. 99.99999% of users, use a  different browser.
